<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Secret_calc</class>
 <widget class="QWidget" name="Secret_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>431</width>
     <height>421</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="normal_calc">
    <attribute name="title">
     <string>Normal Calculator</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="shortcut">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="shortcut">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="shortcut">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="shortcut">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="shortcut">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="shortcut">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="shortcut">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="shortcut">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="shortcut">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="shortcut">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_dot">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="shortcut">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_plus">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>220</y>
       <width>71</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="shortcut">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_minus">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>71</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="shortcut">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_equal">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_divide">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>71</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="shortcut">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_del">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(234, 14, 14);</string>
     </property>
     <property name="text">
      <string>DEL</string>
     </property>
     <property name="shortcut">
      <string>Backspace</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_times">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="shortcut">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_clear">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>111</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="shortcut">
      <string>Backspace, Backspace, Backspace</string>
     </property>
    </widget>
    <widget class="QPushButton" name="norm_percent">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>71</width>
       <height>47</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="shortcut">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="add_calc">
    <attribute name="title">
     <string>Addional Calculators</string>
    </attribute>
    <widget class="QPushButton" name="data_conv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 91, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ziad.png</normaloff>ziad.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>89</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>data convertor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="discount">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 91, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ziad.png</normaloff>ziad.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>89</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>discount</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
